cluster_experiments/power_config.py

```diff
@@ -29,7 +31,7 @@
 )
 
 
-@dataclass
+@dataclass(frozen=True)
```

Comment:
what does this do?

Reply:
This makes the object immutable and hashable.... that means that we can have a set of configs or they can be the key of a dictionary. On the other hand, people can't change the arguments of the class once instantiated